Tasks/DockerV2/dockerpush.ts

```diff
@@ -62,12 +66,28 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
         imageNames = connection.getQualifiedImageNamesFromConfig(repositoryName);
     }
 
+    const dockerfilepath = tl.getInput("dockerFile", true);
+    const dockerFile = findDockerFile(dockerfilepath);
+    if (!tl.exist(dockerFile)) {
+        throw new Error(tl.loc('ContainerDockerFileNotFound', dockerfilepath));
+    }
+
     // push all tags
     let output = "";
-    let promise = pushMultipleImages(connection, imageNames, tags, commandArguments, (commandOut) => output += commandOut);
-    
+    let outputImageName = "";
+    let digest = "";
+    let promise = pushMultipleImages(connection, imageNames, tags, commandArguments, (image, commandOutput) => {
+        output += commandOutput;
+        outputImageName = image;
```

Comment:
What about the other images?

Reply:
I have moved pushImageDetails code inside onCommandOut which is called individually for each of the images inside a for loop.
Have verified for multiple images